pkg/monitoring/top/top.go

```diff
@@ -91,6 +94,7 @@ func (t *Top) startMeasureProcessLoad(interval time.Duration) {
 					PID:        p.PID,
 					Command:    p.Command,
 					Identifier: p.Name,
+					Load1:      ring.New(60),
```

Comment:
you should divide ring.New arg with `interval`

Reply:
that is correct 👍 
done